### PR TITLE
Fixing warnings raised by `make lesson-check-all` command.

### LINKED ISSUE #321: Fix issues raised by make lesson-check
```
 _episodes/03-organization.md:87:    look for uses of `FIXME`.
./_episodes/04-formatting.md:178: Unknown or missing code block type language-html
./_episodes/04-formatting.md:244: Unknown or missing code block type language-yaml
./_episodes/05-rmarkdown-example.md:104: Unknown or missing blockquote type None
./_episodes/05-rmarkdown-example.md:123: Unknown or missing blockquote type None
./_episodes/06-style-guide.md:198: Unknown or missing blockquote type None
./_episodes/06-style-guide.md:202: Unknown or missing blockquote type None
make: *** [lesson-check] Error 1
Makefile:135: recipe for target 'lesson-check' failed
```

### PR DESCRIPTION
**Pull request by vinisalazar**
Fixes relative to #321. Discussion is ensuing over there.

Changes in this PR include:

- Shorten line lengths
- Modify code block tags
- Add `None` as a valid blockquote tag
- Add a link to the Curriculum Development Handbook in `_includes/links.md`

Best,
V

Note: this will fix #321

### CODE REVIEW

File: bin/lesson_check.py [hunk 1]
```diff
@@ -69,11 +70,13 @@
...
     'solution',
     'testimonial',
-    'warning'
+    'warning',
+    None
 }
 
 # What kinds of code fragments are allowed?
 # Below we allow all 'language-*' code blocks
+# These can be configured in assets/css/lessons.scss
```
Comment (maxim-belkin) [inline bin/lesson_check.py:79]: same as before -- please undo this change.

**Comment by vinisalazar** [comment]
Hi @maxim-belkin, thank you for the review. I applied the changes you've requested. I needed to reset a few commits so I did that and force pushed a new commit with the requested changes. Please let me know if there are any more problems.

File: _episodes/07-checking.md [hunk 2]
```diff
@@ -52,6 +52,7 @@ Run `make` on its own to get a full list of commands.
 
 In order to use Jekyll and/or the checking script,
 you may need to install it and some other software.
-The [setup instructions]({{ page.root }}{% link setup.md %}#optional-jekyll-setup-for-lesson-development) explain what you need and how to get it.
```
Comment (maxim-belkin) [inline _episodes/07-checking.md:55]: `(Optional) Jekyll Setup for Lesson Development` title was renamed to `Jekyll Setup for Lesson Development` in 24dc11181429af418433216cf560df000ae83190 and then to `Setup for local rendering of the lessons (optional)` in  cbde2cb51b75b6d8e0765e5f3732ba96bf8da636. I propose to change `#optional-jekyll-setup-for-lesson-development` to `#setup-for-local-rendering-of-the-lessons-optional` or set the ID in the setup.md manually to something shorter.

Comment (vinisalazar) [reply _episodes/07-checking.md:55]: I attempted to fix this with c3aac9f. Let me know what you think.

Comment (maxim-belkin) [reply _episodes/07-checking.md:55]: Looks good to me! Thanks!

**Review by maxim-belkin — APPROVED** [review]
👍🏻  Thanks for your contribution, Vini!

**Comment by maxim-belkin** [comment]
@zkamvar @fmichonneau, the PR looks good to me. The only issue that I can't comment on is the changes to the _episodes/05-rmarkdown-example.md file -- it's autogenerated so I'm not sure if it makes sense to change it or not.

**Review by zkamvar — CHANGES_REQUESTED** [review]
It looks good so far and I especially like the zeal of the author converting long URLs to reference links. There were a couple of improvements I suggested in my review.

Regarding @maxim-belkin's comment about episode 5: he is correct that it should be edited in the `_episodes_rmd/` folder. Also: because we have an R Markdown file in this lesson, we might need to switch to using the 'main' branch as the default so that it doesn't have to be manually rendered every time we need to update that file, but I think we can do that after this PR goes through.

**Comment by vinisalazar** [comment]
Hi @zkamvar, I applied the requested changes. Please, let me know what you think.

Best,
V

File: _includes/links.md [hunk 3]
```diff
@@ -10,6 +10,7 @@
 [cran-checkpoint]: https://cran.r-project.org/package=checkpoint
 [cran-knitr]: https://cran.r-project.org/package=knitr
 [cran-stringr]: https://cran.r-project.org/package=stringr
+[curriculum-handbook]: https://carpentries.github.io/curriculum-development/
```
Comment (maxim-belkin) [inline _includes/links.md:13]: I've just noticed that you added `curriculum-handbook` link here -- please remove it: changes to this file have to be made in carpentries/styles repo.

Comment (vinisalazar) [reply _includes/links.md]: Thank you @maxim-belkin. I added the link directly to `index.md` to prevent a "Line too long" warning.

File: index.md [hunk 4]
```diff
@@ -30,10 +33,15 @@ please see [the setup instructions]({{ page.root }}{% link setup.md %}).
...
 > 8.  Put solutions inside challenges using nested blockquotes.
 > 9.  File issues and template fixes in the [styles repository][styles],
 >     and enhancements to this documentation in this one.
 {: .checklist}
 
 {% include links.md %}
+
+[curriculum-handbook]: https://carpentries.github.io/curriculum-development/
+[tech-intro]: https://carpentries.github.io/curriculum-development/technological-introductions.html
+
```
Comment (maxim-belkin) [inline index.md:47]: Currently, the `{% include links.md %}` line must be the last non-empty line in the file (see `check_reference_inclusion` function in `bin/lesson_check.py`.

Comment (vinisalazar) [reply index.md]: Thank you @maxim-belkin. Pardon my inexperience, but is that documented anywhere? Could be related to carpentries/styles#419.

**Review by zkamvar — APPROVED** [review]
Looks good to me, as was mentioned before, the change made to episode 5 will disappear because it needs to be made in `_episodes_rmd/`, but it's a single line, so we can address that later.
